sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/filesystem/InputComponentStore.java

```diff
@@ -145,21 +142,18 @@ private void addToLanguageCache(DefaultInputFile inputFile) {
   public InputComponentStore put(InputDir inputDir) {
     DefaultInputDir dir = (DefaultInputDir) inputDir;
     inputDirCache.put(dir.moduleKey(), inputDir.relativePath(), inputDir);
+    // FIXME an InputDir can be already indexed by another module
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)